src/plugins/visualizations/public/legacy/build_pipeline.ts

```diff
@@ -507,39 +453,45 @@ export const buildPipeline = async (
   }
   pipeline += '| ';
 
-  // request handler
-  if (vis.type.requestHandler === 'courier') {
-    pipeline += `esaggs
+  if (vis.type.toAST) {
```

Comment:
I always forget what naming convention we decided on for these. `toAST`? `toExpression`? `toExpressionAst`?

I think it's all over the place now, but might be good to choose something we are okay with applying everywhere before all of the vis renderers get written.

Reply:
i agree, you are using `toExpression` ?

Reply:
aggs are using `toExpressionAst`